components/Register/RegisterPage.js

```diff
@@ -0,0 +1,163 @@
+                                margin="normal"
+                                required
+                                fullWidth
+                                id="email"
+                                label="Email Address"
+                                name="email"
+                                autoComplete="email"
+                                autoFocus
+                                value={userEmail}
+                                onChange={(e) => setEmail(e.target.value)}
+                            />
+                            <TextField
+                                variant="outlined"
+                                margin="normal"
+                                required
+                                fullWidth
+                                id="first_name"
+                                label="First Name"
+                                name="text"
+                                autoComplete="text"
+                                value={userFirstName}
+                                onChange={(e) => setFirstName(e.target.value)}
+                            />
+                            <TextField
+                                variant="outlined"
+                                margin="normal"
+                                required
+                                fullWidth
+                                id="last_name"
+                                label="Last Name"
+                                name="text"
+                                autoComplete="text"
+                                value={userLastName}
+                                onChange={(e) => setLastName(e.target.value)}
+                            />
+                            <TextField
+                                variant="outlined"
+                                margin="normal"
+                                required
+                                fullWidth
+                                name="password"
+                                label="Password"
+                                type="password"
+                                id="password"
+                                autoComplete="current-password"
+                                value={userPassword}
+                                onChange={(e) => setPassword(e.target.value)}
+                            />
+                            <TextField
+                                variant="outlined"
+                                margin="normal"
+                                required
+                                fullWidth
+                                id="phone_number"
+                                label="Phone Number"
+                                name="number"
+                                autoComplete="text"
+                                value={userPhonerNumber}
+                                onChange={(e) => setphoneNumber(e.target.value)}
+                            />
```

Comment:
you are repeating `TextField` too many times, find a way to way over the values that you to create these elements